src/pudl/metadata/classes.py

```diff
@@ -2028,6 +2057,7 @@ def to_sql(
                     metadata,
                     check_types=check_types,
                     check_values=check_values,
+                    dialect=dialect,
```

Comment:
Note that there are a number of resources (all the hourly ones which are currently parquet-only, so in CEMS, EIA-930, FERC-714, GridPathRAToolkit) that have `create_database_schema==False` but for the DuckDB output it seems like we probably do want all of those tables included in the unified database file since it's compressed and performant and would really be an all-in-one resource.

Reply:
Ah good reminder. I'll add some logic to `to_sql` to only filter out those tables if we're using sqlite.